src/utils/AssayGeneratorMetabo.cpp

```diff
@@ -443,7 +441,7 @@ class TOPPAssayGeneratorMetabo :
           sirius_workspace_directory = String(sw_dir.absolutePath());
 
           // move tmp folder to new location
-          bool copy_status = File::copyDirRecursively(tmp_dir.toQString(), sirius_workspace_directory.toQString());
+          bool copy_status = File::copyDirRecursively(sirius_tmp.getTmpDir().toQString(), sirius_workspace_directory.toQString());
```

Comment:
same here: moving possible?
If yes, you could set the `sirius_tmp.getTmpDir()` to empty so the DTor does not invoke deletion of a non-existant directory. Or am I missing something?

Reply:
Same as above. I think moving would be possible, but debug would then reference a non-existing path since the output had been moved previously. 

I copied to have the debug output in the same place. 

If the debug output is not needed it is removed afterwards after the copy step. 

If "-debug 3" would be used, the path before the move would be referenced see d'tor SiriusAdapterAlgorihm.